install.sh

```diff
@@ -1,35 +1,47 @@
+#!/bin/bash
 # installation of the linux drivers
-# the installation path must be not changed
-# we hardcoded the path to mitigate problems with LD_LIBRARY_PATH 
-INSTALLPATH=/usr/local
-sudo mkdir -p $INSTALLPATH/bin
-sudo mkdir -p $INSTALLPATH/lib
-sudo mkdir -p $INSTALLPATH/include
+# the installation path must not be changed
+# we hardcoded the path to mitigate problems with LD_LIBRARY_PATH
+if [[ $EUID -ne 0 ]]; then
+   echo "Script must be run with root privileges (\"sudo\")" 2>&1
+   exit 1
+fi
+
+INSTALL_PATH=/usr/local
+
+mkdir -p $INSTALL_PATH/bin
+mkdir -p $INSTALL_PATH/lib
+mkdir -p $INSTALL_PATH/include
+
 echo install library
-sudo cp lib/libbiovisionMultiDaq.so $INSTALLPATH/lib/libbiovisionMultiDaq.so
+cp lib/libbiovisionMultiDaq.so $INSTALL_PATH/lib/libbiovisionMultiDaq.so
 if [ $? -ne 0 ]; then
-   echo "Error occurred!"
+   echo "Error installing library occurred!"
    exit 1
 fi
-echo install headerfiles
-sudo cp include/biovisionMultiDaq.h $INSTALLPATH/include/biovisionMultiDaq.h
+
+echo install header files
+cp include/biovisionMultiDaq.h $INSTALL_PATH/include/biovisionMultiDaq.h
 if [ $? -ne 0 ]; then
-   echo "Error occurred!"
+   echo "Error installing header files occurred!"
    exit 1
 fi
-sudo killall biovisionSharedMemDriver
+
 echo install sharedMemdriver
-sudo cp bin/biovisionSharedMemDriver $INSTALLPATH/bin/biovisionSharedMemDriver
+killall biovisionSharedMemDriver 1>&- 2>&-
```

Comment:
this surpresses `killall` error messages, e.g. when bioVisionSharedDriver was not running anyway.